src/alph_puzzle.c

```diff
@@ -823,7 +823,7 @@ void ov110_021E6904(AlphPuzzleData *data, int a1) {
     }
     
     u32 width = FontID_String_GetWidth(4, data->quitText, 0);
-    AddTextPrinterParameterized2(data->window, 4, data->quitText, (48 - width) / 2, 4, 0xFF, ov110_021E6DA4[a1], NULL);
+    AddTextPrinterParameterizedWithColor(data->window, 4, data->quitText, (48 - width) / 2, 4, 0xFF, ov110_021E6DA4[a1], NULL);
```

Comment:
may be out of scope, but ensure that the colours are generated using the appropriate macro here (that may have been my job that I forgot to do in my last PR, whoops)

Reply:
and text speed constants